package.json

```diff
@@ -95,6 +95,7 @@
         "is-svg": "4.3.0",
         "jszip": "3.5.0",
         "mldoc": "1.2.7",
+        "remove-accents": "0.4.2",
```

Comment:
remove-accents hasn't been updated for almost 4 years.
And has pending PRs in its repo.

Reply:
It's a single file lib, and basically is a dict. The alternative is heavier and harder to install.
Maybe can dump it into our repo? (`resources/js`?)

Reply:
Putting js files to resources can make it hard to upgrade dependencies, for example, our excalidraw integration is outdated. We can fork it if there's pending PRs that could be useful for us.